packages/data-service/src/data-service.ts

```diff
@@ -1579,7 +1579,8 @@ class DataServiceImpl extends WithLogContext implements DataService {
     debug('connecting...');
     this._isConnecting = true;
 
-    this._logger.info(mongoLogId(1_001_000_014), 'Connecting', {
+    this._logger.info(mongoLogId(1_001_000_014), 'Connecting Started', {
+      connectionId: this._id,
```

Comment:
IIUC this _id is per attempt not per cluster, but that's fine for the use case in mms. LMK if "connectionId" isn't clear, I think its just important to not conflate that with a clusterId of some kind